data-dashboard/pages/signup.tsx

```diff
@@ -90,18 +96,78 @@ function SignUp() {
   });
 
   const [credentials, setCredentials] = useState({
-    name: '',
     email: '',
     password: '',
     confirmPassword: '',
   });
 
+  const [error, setError] = useState({
```

Comment:
Most of the time error states are unnecessary when we are using trpc. Could you please remove this and use the error returned by trpc?
```suggestion
```